### ISSUE TITLE
Gem::InvalidSpecificationException

### ISSUE THREAD

**Issue by ddmud** [weppos/whois-parser#108]
Trying to build & install but get the following when I run

`gem build whois-parser.gemspec`

```
WARNING:  See http://guides.rubygems.org/specification-reference/ for help
ERROR:  While executing gem ... (Gem::InvalidSpecificationException)
    ["lib/whois/parsers/whois.domain.kg.rb"] are not files
```

Sure enough, that file doesn't exist when I `git clone git@github.com:weppos/whois-parser.git`

**Comment by weppos** [comment]
@ddmud out of curiosity, is there a reason you are building from scratch rather than using the packaged version from RUbyGems?

**Comment by weppos** [comment]
I'll have to change the gemspec to generate the file list on the fly, rather than store it entirely.

**Comment by weppos** [comment]
Closed by e9cffacbd2b55ffd230b95dd211718f9eb3e97f7
